core/typeSugar.ml

```diff
@@ -362,7 +362,7 @@ sig
   val bind_rec_annotation : griper
   val bind_rec_rec : griper
 
-  val bind_unsafe_fun_annotation : griper
+(*XXX  val bind_unsafe_fun_annotation : griper *)
```

Comment:
Why leave this?

Reply:
I meant to remove it and missed it.  Thanks.